### ISSUE TITLE
[semantic-issue] Add semantics for sklearn accuracy_score

### ISSUE THREAD

**Issue by osryde** [spangea/Pyra#167]
**Description**
This function computes subset accuracy.
Returns a float.

**Minimal running example**
```
from sklearn.metrics import accuracy_score

y_test = train_test_split(Xdata_scaled, y, test_size=0.25, random_state=5)
lgc = LogisticRegression()
lgc.fit(x_train, y_train)
y_test_pred = lgc.predict(x_test)
accuracy_score(y_test, y_test_pred)
```

**Links to documentation**
https://scikit-learn.org/stable/modules/generated/sklearn.metrics.accuracy_score.html

**Note (optional)**
Notebook: 0-91-accuracy-score-for-4-models-lr-rc-gbc-rfc.py

Example:
```
>>> y_pred = [0, 2, 1, 3]
>>> y_true = [0, 1, 2, 3]
>>> accuracy_score(y_true, y_pred)
0.5
```
